pkg/scaffold/api.go

```diff
@@ -271,12 +284,33 @@ func (api *API) scaffoldV2() error {
 	return nil
 }
 
-// Since we support single group only in v2 scaffolding, validate if resource
-// being created belongs to existing group.
-func (api *API) validateResourceGroup(r *resource.Resource) error {
+// isNewGroup will return true if the group informed is diff of the pre-existents
+func (api *API) isNewGroup(r *resource.Resource) bool {
 	for _, existingGroup := range api.project.ResourceGroups() {
 		if strings.ToLower(r.Group) != strings.ToLower(existingGroup) {
-			return fmt.Errorf("group '%s' is not same as existing group '%s'. Multiple groups are not supported yet.", r.Group, existingGroup)
+			return true
+		}
+	}
+	return false
+}
+
+// validateResourceGroup will return an error if the group cannot be created
+func (api *API) validateResourceGroup(r *resource.Resource) error {
+	if api.resourceExists() {
+		return fmt.Errorf("group '%s', version '%s' and kind '%s' already exists.", r.Group, r.Version, r.Kind)
+	}
+	if api.isNewGroup(r) {
+		if api.project.Version == "1" {
```

Comment:
Hmm... It seems `validateResourceGroup` is used by v2 only, so this won't happen.
And V1 actually only support multi-group structure.

Reply:
Good catch makes sense.